### PR TITLE
[TW-1585] Add a memory allocation troubleshooting entry

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Adds a troubleshooting topic for out-of-memory issues

[Preview](https://tw-1585-add-a-memory-allocation-troubleshooting-entry.learning.postman-beta.com/docs/live-insights/live-insights-troubleshoot/#ecs-memory-issues-troubleshooting)

### CODE REVIEW

File: src/pages/docs/live-insights/live-insights-troubleshoot.md [hunk 1]
```diff
@@ -41,9 +56,43 @@ The following is the complete list of error messages.
...
+
+If the instance runs out of memory, one of the containers may stop operating. If you see instances restarting after an `OutOfMemory` outage, you may wish to either increase the hard limit for the task, or impose a hard limit on the agent. If you're performing the latter, set the **Memory hard limit** value in the container definition to at least 700 MB.
+
+![Memory hard limit](https://assets.postman.com/postman-docs/v10/live-insights-memory-hard-limit-v10-21.jpg)
+
+For more information about debugging, see [How do I troubleshoot OutOfMemory errors in Amazon ECS?](https://repost.aws/knowledge-center/ecs-resolve-outofmemory-errors).
+
+### Container size is too large
+
+If you don't specify a task-level memory limit, then the CLI `ecs add` command sets a request of 300 MB. This means that your task requires an additional 300 MB of memory to be scheduled on an instance.
```
Comment (mgritter) [inline src/pages/docs/live-insights/live-insights-troubleshoot.md:79]:
```suggestion
If you don't specify a task-level memory limit, then the CLI `ecs add` command sets a container-level memory request of 300 MB. This means that your task requires an additional 300 MB of memory to be scheduled on an instance.
```

**Review by mgritter — APPROVED** [review]

File: src/pages/docs/live-insights/live-insights-troubleshoot.md [hunk 2]
```diff
@@ -18,6 +18,21 @@ contextual_links:
 
 As you work with the Postman Live Collections Agent (LCA), you may encounter errors and receive error messages. You may also come across frequently encountered issues. You'll find [solutions](#frequently-asked-questions) to some of these issues below.
```
Comment (jkonrath) [inline src/pages/docs/live-insights/live-insights-troubleshoot.md:19]: You can probably bump up your `updated` date

File: src/pages/docs/live-insights/live-insights-troubleshoot.md [hunk 3]
```diff
@@ -41,9 +56,43 @@ The following is the complete list of error messages.
 | _Traffic capture failure_       | Error in capturing traffic                                                            | Something went wrong while capturing traffic. Please try again.                                                                                                                                                                      |
 | _Something went wrong_          | Something went wrong                                                                  | Something went wrong while starting the Agent. Please try again.                                                                                                                                                                     |
 
+## ECS memory issues troubleshooting
+
+The Live Collections Agent (LCA) needs additional memory to run, so if you have used most of the memory available on an instance, some changes may be necessary.
```
Comment (jkonrath) [inline src/pages/docs/live-insights/live-insights-troubleshoot.md:61]: Flip this and the next sentence around a bit so the memory requirement comes first. "The LCA needs at least 300mb to run... if you have used most memory available, you may need to make changes. Also consider setting an upper limit..." etc.

File: src/pages/docs/live-insights/live-insights-troubleshoot.md [hunk 4]
```diff
@@ -41,9 +56,43 @@ The following is the complete list of error messages.
...
+```
+
+![Insufficient memory](https://assets.postman.com/postman-docs/v10/live-insights-insufficient-memory-v10-21.jpg)
+
+In this case, you may need to reduce the memory or `memoryReservation` parameters on other containers in the task by the 300 MB (or 512 MB) requested by the Postman Live Agent.
+
 ## Frequently asked questions
 
-Following topics can help you get traffic in the right state and get API models show the information you’re looking for.
+Following topics can help you get traffic in the right state and get API models show the information you're looking for.
```
Comment (jkonrath) [inline src/pages/docs/live-insights/live-insights-troubleshoot.md:95]:
```suggestion
The following topics can help you get traffic in the right state and get API models to show the information you need.
```

**Review by jkonrath — COMMENTED** [review]
Looks good! A few little suggestions, and I'm on the wrong account and wrong machine, so I can't approve... 🎄

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/live-insights/live-insights-troubleshoot.md (hunk 3 above)
Comment (nerminamiller-postman) [inline src/pages/docs/live-insights/live-insights-troubleshoot.md:61]:
```suggestion
The Live Collections Agent (LCA) needs at least 300 MB of memory to run. if you’ve used most memory available, you need to make adjustments. Consider specifying an upper limit.
```

File: src/pages/docs/live-insights/live-insights-troubleshoot.md [hunk 5]
```diff
@@ -41,9 +56,43 @@ The following is the complete list of error messages.
 | _Traffic capture failure_       | Error in capturing traffic                                                            | Something went wrong while capturing traffic. Please try again.                                                                                                                                                                      |
 | _Something went wrong_          | Something went wrong                                                                  | Something went wrong while starting the Agent. Please try again.                                                                                                                                                                     |
 
+## ECS memory issues troubleshooting
+
+The Live Collections Agent (LCA) needs at least 300 MB of memory to run. if you’ve used most memory available, you need to make adjustments. Consider specifying an upper limit.
+
+Because the agent requires at least 300 MB of memory to operate, consider setting an upper limit.
```
Comment (nerminamiller-postman) [inline src/pages/docs/live-insights/live-insights-troubleshoot.md:63]:
```suggestion
```

File: src/pages/docs/live-insights/live-insights-troubleshoot.md [hunk 6]
```diff
@@ -41,9 +56,42 @@ The following is the complete list of error messages.
 | _Traffic capture failure_       | Error in capturing traffic                                                            | Something went wrong while capturing traffic. Please try again.                                                                                                                                                                      |
 | _Something went wrong_          | Something went wrong                                                                  | Something went wrong while starting the Agent. Please try again.                                                                                                                                                                     |
 
+## ECS memory issues troubleshooting
+
+The Live Collections Agent (LCA) needs at least 300 MB of memory to run. if you’ve used most memory available, you need to make adjustments. Consider specifying an upper limit.
+
+
```
Comment (nerminamiller-postman) [inline src/pages/docs/live-insights/live-insights-troubleshoot.md:63]:
```suggestion
```

**Review by mark-anthony-postman — APPROVED** [review]
Latest changes look good!

**Review by DawallAZllon91 — COMMENTED** [review]
Ejecutando

**Pull request merged**